### ISSUE TITLE
Add indication of computation progress

### ISSUE THREAD

**Issue by karel-brinda** [OndrejSladky/kmercamel#82]
It would be great to have some indication of the progression of the computation. Currently there's none, which is a bit frustrating.

See eg the output of `bwa index` for a good example.

**Comment by PavelVesely** [comment]
I agree that this would be very useful while also not requiring a lot of work
